src/jobs/index.ts

```diff
@@ -74,39 +86,44 @@ export async function handleJobRoute(
   tx.finish();
 }
 
-export const opts = {
-  schema: {
-    body: {
-      type: 'object',
-      required: ['message'],
-      properties: {
-        message: {
-          type: 'object',
-          required: ['data'],
-          properties: {
-            data: {
-              type: 'string',
-            },
-          },
-        },
-      },
-    },
-  },
-};
+// Error handling wrapper function for Cron Jobs
+// Additionally handles Auth from Cloud Scheduler
+export async function handleCronJobs(
```

Comment:
Nit: Could we use singular for naming here as well? I feel it makes more sense since we are only handling a single cron job here
```suggestion
export async function handleCronJob(
```